### DISCUSSION TITLE
Forms, dynamically add fields / modifie

### DISCUSSION THREAD

**Discussion by cube4d** [symfony/symfony#53983] [Q&A]
Hi,

i have my Entity Course with the fields Date1, Date2 ....
In my Form i have a select type with Option1 and Option2 .... now i simple just want, when the form is shown the Date2 Field is not available, first when the user select Option 2 the Field Date 2 shut be added.

I read the examples in the documentation:
https://symfony.com/doc/current/form/dynamic_form_modification.html#dynamic-generation-for-submitted-forms

...and i get it to work like in the example with the ajax at the end. But i just cant figure it out to simple just add one Field?

Hope someone can help me :-)

Thanks

**Comment by norkunas** [comment]
Maybe https://github.com/SymfonyCasts/dynamic-forms will help for you :)
